addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -42,7 +42,7 @@ import {
     isCSSColor,
     convertCSSColorToRgba,
     normalizeCSSColor,
- } from '@web/core/utils/colors';
+} from '@web/core/utils/colors';
```

Comment:
From what I can see, I think your JS linter is misconfigured. In this JS file, I see a lot of changes that aren't related to your task. This shouldn't be the case. Try configuring your linter to check only the lines you've modified.

This has a number of advantages: it makes the reviewer's job easier (he has to check fewer lines of code), and the history of code changes isn't filled with reformatting.